Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedBackingStoreTile.cpp

```diff
@@ -44,15 +44,15 @@ void CoordinatedBackingStoreTile::addUpdate(Update&& update)
 void CoordinatedBackingStoreTile::processPendingUpdates(TextureMapper& textureMapper)
 {
     auto updates = WTFMove(m_updates);
-    auto updatesCount = updates.size();
+    uint64_t updatesCount = updates.size();
```

Comment:
```suggestion
    size_t updatesCount = updates.size();
```
Or we can leave it as `auto`. No need to switch to 64-bit.